fastrtps.repos

```diff
@@ -0,0 +1,18 @@
+repositories:
+    fastcdr:
+        type: git
+        url: https://github.com/eProsima/Fast-CDR.git
+        version: master
+    fastrtps:
+        type: git
+        url: https://github.com/eProsima/Fast-RTPS.git
+        version: 1.8.2
```

Comment:
We should add version numbers to the other repositories too.

Reply:
I guess the other repositories should be at their current latest, right?

Reply:
Release v1.9.1 had then with the explicit versions, check here: https://github.com/eProsima/Fast-RTPS/blob/1.9.x/fastrtps.repos